platforms/hyperledger-indy/configuration/samples/network-indyv3.yaml

```diff
@@ -1,5 +1,9 @@
 ---
+# This is a sample configuration file for hyperledger indy.
+
```

Comment:
Add more details about this sample like this has 3 organizations with x stewards, and can be used for a sample indy network of x nodes